io.github.rfrench3.scopebuddy-gui.yml

```diff
@@ -0,0 +1,70 @@
+id: io.github.rfrench3.scopebuddy-gui
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: scopebuddy_gui.py
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --env=QT_QPA_PLATFORM=wayland;xcb
+# purpose of app is to edit  xdg-config/scopebuddy/scb.conf
+  - --filesystem=xdg-config/scopebuddy:create
+  - --env=QT_LOGGING_RULES=qt.dbus.integration=false
+  - --filesystem=xdg-config/kdeglobals:ro
+
+
+modules:
+  - name: shiboken6
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps shiboken6-6.9.0-cp39-abi3-manylinux_2_28_x86_64.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/45/d3/f6ddef22d4f2ac11c079157ad3714d9b1fb9324d9cd3b200f824923fe2ba/shiboken6-6.9.0-cp39-abi3-manylinux_2_28_x86_64.whl
```

Comment:
hopw will that work on aarch64? Also this runtime use Python 3.12

There is pyside baseapp you should use that.